### PR TITLE
projects/trenchboot-aem.md: add application with giscus system

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

File: docs/projects/trenchboot-aem.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+measurements. These integrity measurements either from SRT or DRT can be used
+for operating system attestation. However, DRT technologies are designed in a
+way to provide the ability to establish a secure environment for integrity
+measurements at any arbitrary point of time instead of relying on the firmware
+which requires machine reset to establish the aforementioned secure
+environment.
+
+The usage of DRT technologies like Intel Trusted Execution Technology (TXT) or
+AMD Secure Startup becomes more and more significant, for example, Dynamic Root
+of Trust for Measurement (DRTM) requirements of Microsoft Secured Core PCs
```
Comment (pietrushnic) [inline docs/projects/trenchboot-aem.md:28]: I think we should [labels](https://spec.commonmark.org/0.30//#link-reference-definitions) or just link, why we keep things in so weird way? Current format cannot be transferred easily to MoU.

```suggestion
of Trust for Measurement (DRTM) requirements of [Microsoft Secured Core PCs](https://some.link)
```

File: docs/projects/trenchboot-aem.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+
+\[3\]: `https://nlnet.nl/project/OpenDRTM/`
+
+\[4\]: `https://www.qubes-os.org/`
+
+\[5\]: `https://blog.invisiblethings.org/2011/09/07/anti-evil-maid.html`
+
+\[6\]: `http://theinvisiblethings.blogspot.com/2009/10/evil-maid-goes-after-truecrypt.html`
+
+\[7\]: `https://sourceforge.net/p/tboot/wiki/Home/`
```
Comment (pietrushnic) [inline docs/projects/trenchboot-aem.md:80]: This is redundant

File: docs/projects/trenchboot-aem.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+
+## The ecosystem of the project
+
+3mdeb has a good relationship with the maintainers of relevant projects which
+will participate in review of the work:
+
+- Marek Marczykowski-Górecki (Invisible Things Lab CTO) - Qubes OS maintainer
+- Andrew Cooper (Citrix) - Xen Hypervisor Maintainer
+- Daniel Kiper (Oracle) - GRUB2 Maintainer
+- Daniel Smith (Apertus Solutions) - TrenchBoot founder and maintainer
```
Comment (pietrushnic) [inline docs/projects/trenchboot-aem.md:287]: Why not say how further review will be continued? That people can use Giscus or that they can contribute directly to this repository.

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: docs/projects/trenchboot-aem.md (hunk 2 above)
Comment (miczyg1) [reply docs/projects/trenchboot-aem.md]: Removed

File: docs/projects/trenchboot-aem.md (hunk 1 above)
Comment (miczyg1) [reply docs/projects/trenchboot-aem.md]: Done for all links using labels

**Review by pietrushnic — APPROVED** [review]

File: docs/projects/trenchboot-aem.md (hunk 3 above)
Comment (miczyg1) [reply docs/projects/trenchboot-aem.md:265]: Done

File: docs/projects/trenchboot-aem.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+
+## Compare your own project with existing or historical efforts
+
+3mdeb is a licensed provider for quality coreboot consulting services for 5
+years. We are well-known in the open-source community for maintaining firmware
+of the PC Engines APU series platform for over 4-years. Delivering high-quality
+firmware releases each month and providing technical support on PC Engines and
+OPNSense forums. 3mdeb embedded systems developers are experienced engineers
+accustomed to operating systems development. Our developers have contributed to
+the [fwupd support for Qubes OS][8]. 3mdeb is also regularly co-organizing
```
Comment (marmarek) [inline docs/projects/trenchboot-aem.md:76]:
```suggestion
the [fwupd support for Qubes OS][qubes_fwupd]. 3mdeb is also regularly co-organizing
```

File: docs/projects/trenchboot-aem.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+of Trust for Measurement (DRTM) requirements of
+[Microsoft Secured Core PCs][ms_sec_core]. In open-source projects, DRTM hasn't
+found its place yet, but that gradually changes. The demand on having firmware
+independent Roots of Trust is increasing and projects that satisfy this demand
+are growing, for instance, [TrenchBoot][tb_org]. TrenchBoot is a framework that
+allows individuals and projects to build security engines to perform launch
+integrity actions for their systems. The framework builds upon Boot Integrity
+Technologies (BITs) that establish one or more Roots of Trust (RoT) from which
+a degree of confidence that integrity actions were not subverted. The project
+has grown a lot thanks to the previous [NLnet NGI0 PET grant][nlnet_opendrtm]
```
Comment (marmarek) [inline docs/projects/trenchboot-aem.md:37]:
```suggestion
has grown a lot thanks to the previous [NLnet NGI0 PET grant][nlnet_open_drtm]
```

**Pull request merged**